examples/simple_example/main.tf

```diff
@@ -21,8 +21,7 @@ provider "google" {
 module "folders" {
   source = "../.."
 
-  parent_id         = var.parent_id
-  parent_type       = var.parent_type
+  parent            = var.parent
```

Comment:
Instead of updating the examples/fixtures, how about we keep them the same and concat parent_type and parent_id to create the example invocation?

Reply:
Make sense, updated